src/components/Exploration/FiltersSubGroup.tsx

```diff
@@ -27,7 +27,7 @@ export function FiltersSubGroup({
         <span>{title}</span>
         <i className={`fas fa-caret-${expanded ? "down" : "left"}`} />
       </button>
-      {expanded && <main>{children}</main>}
+      {expanded && <div>{children}</div>}
```

Comment:
Not sure why `<main>` was used here, but it's a big issue in HTML. `<main>` tag should be a single one in the document.